test/fixtures/render/README.md

```diff
@@ -0,0 +1,92 @@
+# About
+
+These files are for testing `hab plan render` command.
+
+see `hab plan render --help` for full usage instructions.
+
+# Usage
+
+`cd` to `habitat/components/hab`
```

Comment:
FYI, instead of `cd`ing, you can pass `-p hab` to `cargo` and save yourself a bunch of `..`s. For example, in the root:
```
➤ cargo run -p hab plan render --default-toml test/fixtures/render/consul/default.toml test/fixtures/render/consul/config/consul_config.json
```

Reply:
Oh. this is awesome.